packages/ember-testing/lib/test.js

```diff
@@ -467,6 +469,107 @@ EmberApplication.reopen({
   }
 });
 
+if (isEnabled('ember-testing-instances')) {
+  EmberApplication.reopen({
+    /**
+      Builds a new instance of the application and registers test helpers on it.
+
+      Example:
+
+      ```javascript
+      var instance = run(App, 'buildTestInstance');
```

Comment:
I'd be fine with that. Currently chose this approach to keep it similar to how `start-app` works.